# Analysis of the 31 -staked sequences
The analysis has been conducted using the new FTLS software developed in python. 

In [1]:
%run C:\Users\MANU\Funcion_plots.ipynb

print function loaded


<a id="top"></a>
## Contents
### Note: Click on the earthquake line to view the detailed tab.


- **[Summary Table](#Summary-Table)**

**[California earthquakes](#California)**
- [Coalinga 2 May 1983](https://rawcdn.githack.com/Biondini/new_FTLS/60452993a3fbbfdd71a0d7516562ff38d9630fac/Coalinga.html)
- [Morgan Hill 24 Apr 1984](https://rawcdn.githack.com/Biondini/new_FTLS/5cf051c90c905b8f458f9ba9376e2513627061f9/Morgan_Hill.html)
- [Round Valley 23 Nov 1984](https://rawcdn.githack.com/Biondini/new_FTLS/fa132bfd15d6d852ddbb343a32a01674e5b0ff69/Round_Valley.html)
- [Chalfant Valley 21 Jul 1986](https://rawcdn.githack.com/Biondini/new_FTLS/e2b3ccc2239e79e6f57dcca9213f9eb90199bbf4/Chalfant_Valley.html)

- [Loma Pireta 18 Oct 1989](https://rawcdn.githack.com/Biondini/new_FTLS/6e8d89e802e93032f48d1f541b5cd72411688ebe/Loma_Pireta.html)
- [Joshua Tree 23 Apr 1992](https://rawcdn.githack.com/Biondini/new_FTLS/8acfa6d66c7e1da596c8d4667892323d3b8a6979/Joshua_Tree.html)
- [Landers 28 Jun 1992](https://rawcdn.githack.com/Biondini/new_FTLS/cb8bb9e1e9b64769822f9ee4fb880200eb089a6e/Landers.html)
- [Northbridge 17 Jan 1994](https://rawcdn.githack.com/Biondini/new_FTLS/03ce0f7ca43727742fbb73dfb54984c7606df105/Northbridge.html)
- [Hector Mine 16 Oct 1999](https://rawcdn.githack.com/Biondini/new_FTLS/3053dbffd701ee62a1e1e56353ace6eb1ca41cc7/Hector_Mine.html)
- [San Simeon 22 Dec 2003](https://rawcdn.githack.com/Biondini/new_FTLS/82ae5f409d23c747b72f46699074326d3b2432a2/San_Simeon.html)
- [Parkfield 28 Sep 2004](https://rawcdn.githack.com/Biondini/new_FTLS/16ffd36e419a8603d54707532510ed3fc552c30b/Parkfield.html)
- [El Mayor Cucapah 4 Apr 2010](https://rawcdn.githack.com/Biondini/new_FTLS/300def26e1772664f213b1c8c0e9853a4f2c037c/El_Mayor_Cucapah.html?min=1)

**[Japan earthquakes](#Japan)**
- [Tottori 6 Oct 2000](https://rawcdn.githack.com/Biondini/new_FTLS/09583c281ddca4a7db96edfe3c43f35bcf25c485/Tottori.html)
- [Ryukyu 18 Dec 2001](https://rawcdn.githack.com/Biondini/new_FTLS/cf12d2554f3f543a7832205fad0bca1ef98dc5d5/Ryukyu.html)
- [Chuetsu 23 Oct 2004](https://rawcdn.githack.com/Biondini/new_FTLS/0e1e8c6ed474ba66f0e32e45346c48ec5c75e840/Chuetsu.html)
- [Fukuoka 20 Mar 2005](https://rawcdn.githack.com/Biondini/new_FTLS/c28f594606b255c9b4c96fc65acffeac436bc53d/Fukuoka.html)
- [Noto 25 Mar 2007](https://rawcdn.githack.com/Biondini/new_FTLS/5ef83129113d280c5c671c418374fc315c12d3b2/Noto.html)
- [Chuetsu Offshore 16 Jul 2007](https://rawcdn.githack.com/Biondini/new_FTLS/bb5eda7a119891a7a72bcc5a48223840de8b4728/Chuetsu_Offshore.html)
- [Iwate 13 Jun 2008](https://rawcdn.githack.com/Biondini/new_FTLS/79216cd2e13fa9bfd5186ebce0bf7e8086a7603e/Iwate.html)
- [Numata, Gunma 25 feb 2013](https://rawcdn.githack.com/Biondini/new_FTLS/ab767ecac12e03f424df2fd97635a8bc23329cc0/Numata-Gunma.html)
- [Awaji Island 12 Apr 2013](https://rawcdn.githack.com/Biondini/new_FTLS/69214c700ff3f314ae0b1729009442aea13f685d/Awaji_Island.html)
- [Nagano 22 Nov 2014](https://rawcdn.githack.com/Biondini/new_FTLS/6e9f75bebb7b747b6756d05015838641cd229ceb/Nagano.html)
- [Fukushima 21 Nov 2016](https://rawcdn.githack.com/Biondini/new_FTLS/a271ffc48a4dc7927a3605733c382f0b0be7d025/Fukushima.html)

**[Italy earthquakes](#Italy)**
- [L'Aquila 6 Apr 2009](https://rawcdn.githack.com/Biondini/new_FTLS/31d2211bb464800a66277a8af7f23806c1df1fde/LAquila.html)

**[Alaska earthquakes](#Alaska)**
- [Denali 3 Nov 2002](https://rawcdn.githack.com/Biondini/new_FTLS/d657d39898143c671047d7f5ac71e68216cb5e84/Denali.html)

**[Unanalyzable earthquakes](#Unanalyzable)**
- [Sperstition Hills 24 Nov 1987 (California)](#Superstition-Hills)
- [Honeydew 17 Aug 1991 (California)](#Honeydew)
- [Eureka Valley 17 May 1993 (California)](#Eureka-Valley)
- [Miyazaki 28 Aug 2014 (California)](#Miyazaki)

# Summary Table
<a id="Summary-Table"></a>
[Index](#top)

#### List of the 27 analyzable staked sequences. 
For each earthquake, information is provided about the date, magnitude, reference b-value (b_ref), the maximum b% estimated in the first 3 months following the mainshock (b-Max90), as well as the strike, dip, and rake of the fault plane, and the relative style of faulting according to the Frolich diagram.

In [2]:
summary_plot()

| Earthquake       | Date                       |   Magnitude |   b_ref |   b-Max90 |   Strike |   Dip |   Rake | Fault type   |
|:-----------------|:---------------------------|------------:|--------:|----------:|---------:|------:|-------:|:-------------|
| Coalinga         | 1983-05-02 23:42:38.060    |        6.7  |    0.75 |   136.323 |      320 |    30 |     87 | TH           |
| Morgan Hill      | 1984-04-24 21:15:18.760000 |        6.2  |    0.81 |   101.752 |      333 |    76 |    179 | SS           |
| Round Valley     | 1984-11-23 18:08:25.360000 |        6.1  |    0.69 |   160.325 |      221 |    82 |    -43 | UN           |
| Chalfant Valley  | 1986-07-21 14:42:26        |        6.4  |    0.67 |   217.43  |      149 |    60 |   -163 | SN           |
| Loma Pireta      | 1989-10-18 00:04:15.190000 |        6.9  |    0.98 |   151.084 |      123 |    71 |    128 | UN           |
| Joshua Tree      | 1992-04-23 04:50:23.230000 |        6.1  |    0.97 |   135.638 |      171 |    89 |   -177 | SS           |
| Landers          | 1992-06-28 11:57:34.130000 |        7.3  |    0.93 |   172.274 |      341 |    70 |   -172 | SS           |
| Northbridge      | 1994-01-17 12:30:55.390000 |        6.7  |    0.91 |   137.901 |      130 |    53 |    111 | TH           |
| Hector Mine      | 1999-10-16 09:46:44.460000 |        7.1  |    0.84 |   222.947 |      336 |    80 |    174 | SS           |
| San Simeon       | 2003-12-22 19:15:56.240000 |        6.5  |    0.9  |   131.802 |      296 |    32 |     88 | TH           |
| Parkfield        | 2004-09-28 17:15:24.250000 |        5.97 |    0.82 |   121.887 |      321 |    72 |   -178 | SS           |
| El Mayor Cucapah | 2010-04-04 22:40:42.360000 |        7.2  |    0.91 |   162.139 |      313 |    88 |   -174 | SS           |
| Tottori          | 2000-10-06 04:30:17.940000 |        7.3  |    0.72 |   229.377 |      331 |    83 |      1 | SS           |
| Ryukyu           | 2001-12-18 04:02:59.360000 |        7.3  |    0.92 |   133.56  |      204 |    59 |    -53 | SN           |
| Chuetsu          | 2004-10-23 09:34:05.690000 |        6.5  |    0.76 |   137.81  |      215 |    62 |     91 | TH           |
| Fukuoka          | 2005-03-20 01:53:40.320000 |        7    |    0.83 |   137.563 |      302 |    88 |    -14 | SS           |
| Noto             | 2007-03-25 00:41:57.910000 |        6.9  |    0.71 |   137.089 |       34 |    40 |    108 | TH           |
| Chuetsu Offshore | 2007-07-16 01:13:22.550000 |        6.8  |    1.06 |   117.505 |      225 |    60 |     94 | TH           |
| Iwate            | 2008-06-13 23:43:45.360000 |        7.2  |    0.84 |   182.838 |       17 |    42 |     87 | TH           |
| Numata, Gunma    | 2013-02-25 07:23:53.580000 |        6.3  |    0.72 |   140.832 |      346 |    85 |     -1 | SS           |
| Awaji Island     | 2013-04-12 20:33:17.750000 |        6.3  |    0.66 |   133.205 |      180 |    68 |     98 | TH           |
| Nagano           | 2014-11-22 13:08:17.900000 |        6.7  |    0.75 |   132.316 |       15 |    41 |     49 | TH           |
| Fukushima        | 2016-11-21 20:59:46.890000 |        7.4  |    0.95 |   150.176 |       49 |    35 |    -89 | NR           |
| L'Aquila         | 2009-04-06 01:32:40.400000 |        6.29 |    0.82 |   188.452 |      336 |    42 |    -62 | NR           |
| Denali           | 2002-11-03 22:12:41.518000 |        7.9  |    0.85 |   132.771 |       29 |    82 |     19 | SS           |

<a id="Unanalyzable"></a>
# Unanalyzable earthquakes
[Index](#top)

<a id="Superstition-Hills"></a>
## Superstition Hills (California)
### Date: 24 Nov 1987 13:15:56
### m = 6.6
#### This event is not analyzable because there are no enough events in the post catalog. 
[Index](#top)

<a id="Honeydew"></a>
## Honeydew (California)
### Date: 17 Aug 1991 19:29:40
### m = 6.0
#### This event is not analyzable because there are no enough events in the post catalog. 
[Index](#top)

<a id="Eureka-Valley"></a>
## Eureka Valley (California)
### Date: 17 May 1993 23:20:50
### m = 6.1
#### This event is not analyzable because there are no enough events in the post catalog. 
[Index](#top)

<a id="Miyazaki"></a>
## Miyazaki (Japan)
### Date: 28 Aug 2014 19:14:35
### m = 6.0
#### This event is not analyzable because there are no enough events in the post catalog. 
[Index](#top)